.github/workflows/build-pr.yml

```diff
@@ -31,35 +30,33 @@ jobs:
         python -c "import os;print(os.environ['GITHUB_REF'].split('/')[2])" > pr_number.txt
         echo "PR_NUMBER=$(cat pr_number.txt)" >> $GITHUB_ENV
         echo "GRADIO_VERSION=$(python -c 'import requests;print(requests.get("https://pypi.org/pypi/gradio/json").json()["info"]["version"])')" >> $GITHUB_ENV
-    - name: Build and publish pr package
+    - name: Build pr package
       run: |
-        export AWS_ACCESS_KEY_ID=${{ secrets.PR_DEPLOY_KEY }}
-        export AWS_SECRET_ACCESS_KEY=${{ secrets.PR_DEPLOY_SECRET }}
-        export AWS_DEFAULT_REGION=us-east-1
         echo ${{ env.GRADIO_VERSION }} > gradio/version.txt
         cd ui
         pnpm i --frozen-lockfile
         pnpm build
         cd ..
         python3 -m build -w
-        aws s3 cp dist/gradio-${{ env.GRADIO_VERSION }}-py3-none-any.whl s3://gradio-builds/${{ github.sha  }}/
       env:
         NODE_OPTIONS: --max_old_space_size=8192
-    - name: Install Hub Client Library
-      run: pip install huggingface-hub
+    - name: Upload wheel
+      uses: actions/upload-artifact@v3
+      with:
+        name: gradio-${{ env.GRADIO_VERSION }}-py3-none-any.whl
+        path: dist/gradio-${{ env.GRADIO_VERSION }}-py3-none-any.whl
     - name: Set up Demos
       run: python scripts/copy_demos.py https://gradio-builds.s3.amazonaws.com/${{ github.sha }}/gradio-${{ env.GRADIO_VERSION }}-py3-none-any.whl
-    - name: Upload kitchen sink to spaces
+    - name: Upload all_demos
```

Comment:
Uploading all_demos as well so that we deploy the code from the branch/fork

Reply:
Nice